src/backend/mailbox_manager/api/viewsets.py

```diff
@@ -38,6 +39,15 @@ class MailDomainViewSet(
     lookup_field = "slug"
     queryset = models.MailDomain.objects.all()
 
+    def get_permissions(self):
+        """User only needs to be authenticated to list accesses"""
```

Comment:
- unrelated people should not be able to list domain's accesses
- owners, admins and viewers (if we stick to this notion) should list accesses
- about viewers, I think you're right, it might be a good thing for them to list accesses. One use that comes to my mind is knowing which email/person to contact in order to create a new mail on your domain (if your team don't make it clear, at least La Régie wil)